### DISCUSSION TITLE
Adaptive Card throwing "Card - access it on https://go.skype.com/cards.unsupported" error when using the Share to Outlook in Microsoft Teams

### DISCUSSION THREAD

**Discussion by hirushanlk** [microsoft/AdaptiveCards#8932] [Q&A]
- I have an adaptive card sent to a Microsoft Teams channel via an incoming webhook using a script in the pipeline for a given number of input variables. 
- Within Teams, the card preview with the expected contents is available. 
- When trying to share the card as an email via the "share to Outlook" option, it throws the following error. 
- I used schema version 1.0 and tried with 1.2 as well. 
- Within the card, Text block, ColumnSet, Column, Image, FactSet and Action.OpenUrl are used as card elements.

![Screenshot 2024-06-11 103138](https://github.com/microsoft/AdaptiveCards/assets/155649986/796d0e8c-3bb1-4f3b-8fb1-5fa3894b5c24)

**Comment by Adrianotiger** [comment]
This error is still present and is present also in Teams, not only Outlook.
If, in Teams, you search for the title, you will see this error message also in there.
<img width="522" height="258" alt="image" src="https://github.com/user-attachments/assets/455aa4ee-a921-4f06-9eaf-677091859c61" />
